js/projects.js

```diff
@@ -91,6 +92,38 @@ async function append_projects_recursively(projectId, order, parentProjectStats,
     .finally(() => {checkFilterButtons(--download_queue_length)})
 }
 
+async function append_important_recursively(buildTypeId, buildTypeOrder, parentProjectStats, parentProjectIds) {
+
+    if (!parentProjectStats) {
+        parentProjectStats = []
+        parentProjectIds = []
+    }
+
+    fetch(`${teamcity_base_url}/app/rest/buildTypes/id:${buildTypeId}?`, {
+        headers: {
+            'Accept': 'application/json',
+        },
+        credentials: 'include',
+        priority: 'high',
+    },this)
+    .then((result) => result.json())
+    .then((buildType) => {
+        checkFilterButtons(++download_queue_length)
+        let project = []
+        project.parentProjectId = 'important'
+        project.id = 'important'

```

Comment:
The project.id is what you see when you hover. You could (ab)use the project.id to add something like: "These are the build types that are defined under 'important_buildtypes' in the Project Selection screen.".